man/systemd.exec.xml

```diff
@@ -681,80 +681,59 @@
         that the screen and scrollback buffer is cleared. Defaults to
         <literal>no</literal>.</para></listitem>
       </varlistentry>
+
       <varlistentry>
         <term><varname>SyslogIdentifier=</varname></term>
-        <listitem><para>Sets the process name to prefix log lines sent
-        to the logging system or the kernel log buffer with. If not
-        set, defaults to the process name of the executed process.
-        This option is only useful when
-        <varname>StandardOutput=</varname> or
-        <varname>StandardError=</varname> are set to
-        <option>syslog</option>, <option>journal</option> or
-        <option>kmsg</option> (or to the same settings in combination
-        with <option>+console</option>).</para></listitem>
+        <listitem><para>Sets the process name ("<command>syslog</command> tag") to prefix log lines sent to the logging
+        system or the kernel log buffer with. If not set, defaults to the process name of the executed process.  This
+        option is only useful when <varname>StandardOutput=</varname> or <varname>StandardError=</varname> are set to
+        <option>journal</option>, <option>syslog</option> or <option>kmsg</option> (or to the same settings in
+        combination with <option>+console</option>) and only applies to log messages written to STDOUT or
+        STDERR.</para></listitem>
```

Comment:
Some time back we agreed on lowercase stdin/stderr/stdout and "standard input, output, and error", as appropriate. See 6db2742802b70938f0f2d373110ed734d4fb9813.